pkg/minishift/addon/manager/addon_manager.go

```diff
@@ -358,5 +371,29 @@ func compareOpenshiftVersions(openShiftVersion, requiredOpenshiftVersion string)
 		return fmt.Errorf("\nAddon does not support OpenShift version %s. "+
 			"You need to use a version %s", openShiftVersion, requiredOpenshiftVersion)
 	}
+
+	return nil
+}
+
+func addVarDefaultsToContext(context *command.ExecutionContext, varDefaults []string) error {
+	for _, name := range varDefaults {
+		match, _ := regexp.Match(".*=.*", []byte(name))
+		if !match {
+			return errors.New(fmt.Sprintf("Var-Defaults variable need to be specified in the format of <key>=<value>.'%s' is not.", name))
```

Comment:
Should be either "needs to be" or "must be". There should also be a space between the period and single quote containing the `%s` replacement.

Reply:
Done